orc8r/cloud/go/services/accessd/servicers/internal_servicers/accessd.go

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
accessd.go is recognised as a new file here. Please use mv command to move existing files instead of deleting and adding a new file with same contents.